src/vmm/intel/vmx.c

```diff
@@ -2041,6 +2042,7 @@ vmx_exit_process(struct vmx *vmx, int vcpu, struct vm_exit *vmexit)
 		 */
 		gpa = vmcs_gpa(vcpu);
 		if (vm_mem_allocated(vmx->vm, gpa) ||
+                    bootrom_contains_gpa(gpa) ||
```

Comment:
I cannot articulate why, exactly, but I'm concerned with this particular change.  It feels like a layering violation.